gemini/gemini_client.py

```diff
@@ -41,13 +45,20 @@ def gemini_recipe_chat(cuisine, cost, nutrition):
                 response_schema=Recipe
             )
         )
+        # Parse the gemini response to get the contents of the list
         recipe_list = response.parsed.recipes
-        # print(recipe_list)
+
+        # Log the response from gemini
+        logging.info(recipe_list)
     
+    except ValueError as e:
+        logging.error(f'Value error occured: {e}')
     except Exception as e:
```

Comment:
Put this at the  end. `except Exception` will catch all exceptions including requests.exceptions.ConnectionError and  json.JSONDecodeError so the two except blocks below this one will never run